_posts/2018-02-02-rebble-web-services.md

```diff
@@ -0,0 +1,131 @@
+Rebble, it's more complicated: you can expect more, but probably not everything.
+
+Timeline functions as a web API, where other servers push timeline updates to Pebble servers, which
+then forward those updates to your watch. We can (and will) rebuild the servers that pass timeline
+updates along to your watch. We will also automatically alter apps that skipped the server and
+just pushed directly into your timeline (there were many) so that they hit our servers instead of
+Pebble's. What we _cannot_ do, however, is alter app developers' servers so they hit us instead of
+Pebble. We will provide them with new endpoints to hit before the time comes, and hope that they
+get the memo to update.
+
+Finally, Timeline updates will become slower. While Timeline is today sometimes able to update
+immediately when a service sends you a pin or notification, for technical reasons it is impossible
+for any entity other than Fitbit to provide that service. Using the Rebble timeline service, updates
+will instead occur every fifteen minutes.
+
+## Weather
+
+Pebble uses the weather in a couple of places: there's the official weather app, and there are the
+sunrise and sunset pins in the Timeline. Today, Pebble fetches this information from The Weather
+Channel. We will replace this with information from the Dark Sky API. Unfortunately, this API (like
+all reliable, high-quality, global weather APIs) costs
+us some money to access, so we will be providing weather access as a **$1/month** perk on Patreon.
+Paying that dollar will enable full weather functionality, and help support the service for
+everyone. If you do _not_ donate on Patreon, the system weather app and timeline pins will not
+appear. However, all third-party watchfaces will continue displaying the weather without
+interruption for as long as the APIs they use continue to function.
+
+## Dictation
+
+Unfortunately, dictation is already broken when using iPhones running iOS 11+ with either a Pebble 2
+or Pebble Time Round. There is nothing that we can do to fix this. For Android users, as well as
+Pebble Time and Pebble Time Steel users, we will be replacing Pebble's Nuance dictation with Google
+Cloud Speech. This, again, will cost us some money each time you use it. In order to run this
+sustainably, we will require that you pledge at least **$2/month** on Patreon in order to access
+dictation (which will also include weather).
+
+## iOS SMS and Email replies/actions
+
+Pebble had agreements in place with several carriers and service providers in order to provide the
+ability to reply to text messages on iOS. Unfortunately, there is no way that we can provide this
+service. Pebble also provided email actions by requesting access to your email account. The privacy
+and security implications of having this access are greater than we are willing to tolerate. As
+such, we will not be providing either iOS or email action support.
```

Comment:
`either iOS or email action support`

makes it sound like email actions on Android might stop working. Is that meant to be something like `iOS SMS or email action support`?

Reply:
Yup! Oops.